geometry_msgs/Pose pose
string color